apache-maven/src/bin/mvn.ps1

```diff
@@ -0,0 +1,158 @@
+<#
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we make this `https`?

Reply:
@yeikel Good point. But why would we do it only in this occurrence? At this moment in time the http link is used in 1,663 places (it redirects properly to https). What would you think about the idea to create a separate pull request to resolve this for the whole codebase? For instance for the `3.9.x` and `master` branch.

Reply:
That's fair. I don't see any issue with that

Reply:
Don't waste your time. This won't happen.